docs/changes/io.ascii/15392.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bugs in ``astropy.io.ascii.qdp`` to allow one "NO" to start a new sector and allow any space-like characters ahead of the "NO".
```

Comment:
I do not see a test for "space-like characters head of the 'NO'" in your diff. Did I miss something?

Reply:
It's already testing with four spaces, but nothing else (see suggestion).